tests/testthat/test-FunctionReporter.R

```diff
@@ -102,13 +101,6 @@ test_that('FunctionReporter Methods Work', {
     expect_true(object = all(igraph::get.vertex.attribute(testPkgGraph)[[1]] %in% testNodeDT$node)
                 , info = "Graph nodes not as expected")
 
-    # Nodes table with coverage and metrics too
-    # TODO: Test that calculate_all_measures and other calculates attach metadata correctly
-    # expect_identical(object = sort(testObj$get_raw_data()$nodes$node)
-    #                  , expected = sort(testNodeDT$node)
-    #                  , info = "Different nodes than expected")
```

Comment:
@bburns632 this is covered by the tests below already, right?

Reply:
yes and more so by https://github.com/UptakeOpenSource/pkgnet/blob/master/tests/testthat/test-NetworkMeasures.R

Reply:
awesome, thank you